pycode/memilio-epidata/memilio/epidata_test/test_epidata_getCaseData.py

```diff
@@ -618,8 +618,7 @@ def test_get_case_data_all_dates_and_split_berlin(self):
         make_plot = False
```

Comment:
Why not use 
```
# activate CoW for more predictable behaviour of pandas DataFrames
pd.options.mode.copy_on_write = True
```
also in the test files?

Reply:
it is activated because the file which is tested activates it. So it would just be redundant here